sova/git/rebase.py

```
@@ -24,6 +42,133 @@ class RebaseResult:
     error: str = ""
 
 
+def _normalize_resolution(text: str) -> str:
+    """Normalize text for consensus comparison."""
+    text = text.replace("\r\n", "\n")
+    lines = text.split("\n")
+    lines = [line.rstrip() for line in lines]
+    result = "\n".join(lines)
+    result = result.rstrip("\n")
+    return result
```

Ruff (0.16.1)

[warning] Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)